### ISSUE TITLE
[Version Support] Non-LTS Ubuntu installation error

### ISSUE THREAD

**Issue by yehyunchoi** [timescale/timescaledb#3854]
I could not find a suitable issue category, so here goes.

# TL;DR;
Is there a way to support non-LTS Ubuntu for timescaleDB repos on packagecloud?

# Description
There is a ubuntu version compatibility issue as I am trying to install timescale onto my ubuntu.
I know the LTS versions are well supported and can apt install no problem. 
However, I needed to use a higher version of Ubuntu for my pc setup due to its wifi adaptor on the motherboard, which will only be supported by kernel higher than 21.x 

While on Ubuntu 21.10 running the following command
`~$ sudo add-apt-repository ppa:timescale/timescaledb-ppa`

Gives the following error
`...`
`Err:6 https://packagecloud.io/timescale/timescaledb/ubuntu impish Release`
`404 Not Found [IP: 54.241.210.185 443]`
`Err:7 http://ppa.launchpad.net/timescale/timescaledb-ppa/ubuntu impish Release`
`404 Not Found [IP: 91.189.95.85 80]`
`...`
Which I've found to be issues with the `impish Release` distribution, and resolved `Err:7` by changing distribution option to `focal`.
But I cannot get `packagecloud.io` Err to work.

Is there a way to support non-LTS Ubuntu for timescaleDB repos on packagecloud?

**Comment by Loquacity** [comment]
This is not a docs issue.

**Comment by yehyunchoi** [comment]
Oh shoot sorry, thanks for moving it here.

**Comment by Loquacity** [comment]
> Oh shoot sorry, thanks for moving it here.

No worries, hope you get the help you need 👍

**Comment by erimatnor** [comment]
@svenklemm do you know how easy it would be to support non-LTS releases?

**Comment by svenklemm** [comment]
> @svenklemm do you know how easy it would be to support non-LTS releases?

Adding this should be straightforward, ideally this is just a change to the build matrix but that depends on the release being added.

**Comment by svenklemm** [comment]
Ubuntu 21.10 is now part of the build matrix and initial packages for 2.5.0 were built.

**Comment by yehyunchoi** [comment]
Sorry for getting late on the reply.
The installation was done successfully for my new setup.
Thank you guys so much !
